content/how-to/dedicated-hosting/create-new-user-on-active-directory-domain/index.md

```diff
@@ -5,8 +5,8 @@ title: Create new user on Active Directory domain
 type: article
 created_date: '2019-12-12'
 created_by: Chad Sterling
-last_modified_date: '2021-07-07'
-last_modified_by: Cat Lookabaugh
+last_modified_date: '2021-09-26'
```

Comment:
need to update the year as well. Might as well just update the date to tomorrows date.

Reply:
updated this year